src/main/java/org/osc/controller/nsc/api/SampleSdnRedirectionApi.java

```diff
@@ -488,4 +494,118 @@ public void close() throws Exception {
         });
     }
 
+    // Support methods for REST API's
+
+    public List<String> getInspectionHooksIds() throws Exception {
+
+        return this.txControl.supports(() -> {
+            return this.utils.txInspectionHookEntities().stream().map(InspectionHookEntity::getHookId)
+                    .collect(Collectors.toList());
+        });
+    }
+
+    public Element updateInspectionPort(InspectionPortEntity inspectionPort) throws Exception {
+
+        InspectionPortEntity inspectionPortEntity = (InspectionPortEntity) getInspectionPort(inspectionPort);
+        this.utils.throwExceptionIfNullElement(inspectionPortEntity);
+
+        if (inspectionPort.getIngressPort().equals(null)
```

Comment:
please add method like we did in the neutron sfc plugin 
org.osc.controller.nsfc.utils.RedirectionApiUtils.throwExceptionIfNullElementAndId(Element, String)